packages/CREATING_PLUGINS.md

```diff
@@ -0,0 +1,68 @@
+# Creating plugins
```

Comment:
Should this documentation not be on the website?

Reply:
In general, I would say that we should not maintain two versions of the same information. At some point, they will always diverge.

To me, this seems information more relevant on the website. In the repo, I would limit the documentation to code documentation and READMEs for structure and providing links

Reply:
Removed.